### PR TITLE
Updates Banner on the Advocacy site

### PR DESCRIPTION
**Pull request by hannahbergam**
Adds the new string and updates all strings (and button color) on the advocacy banner on https://advocacy.code.org/!

Before:
Screenshot 2024-04-15 at 3 08 28 PM

After:
Screenshot 2024-04-15 at 5 39 08 PM

## Warning!!

The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) is in progress. The most critical dates are from April 3 - April 30, 2024. Please consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking anything in these two labs. Even small changes, such as a different button color, are considered significant during this time period. Reach out to the Student Learning team or Curriculum team for more details.

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1608
- Figma: https://www.figma.com/file/hNBxdgZ40KH4UZFG7ZEVo8/AI-Landing-Page-Updates---Spring-2023?type=design&node-id=3703-294&mode=design&t=8zAvbJx2UqAKIBwP-0

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/advocacy.code.org/public/index.haml [hunk 1]
```diff
@@ -68,8 +68,8 @@ video_player: false
       %img{src: "/images/teach-ai-logo.svg", alt: ""}
       .teach-ai-text{style: "text-align: center"}
         %p.heading-sm{style: "color: white;"}
-          =hoc_s(:teach_ai_title)
+          =hoc_s(:ai_promote_literacy)
         %p{style: "color: white; margin-bottom: 0"}
-          =hoc_s(:teach_ai_description)
-      %a.link-button.white.has-external-link{style: "flex: 0 0 auto;", href: "https://teachai.org/toolkit", target: "_blank", rel: "noopener noreferrer"}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/advocacy.code.org/public/index.haml:74]: Let's keep this button white (this is a case of knowing when to override the mock 🥲)

**Review by kelbyhawn — APPROVED** [review]
Looks good, have one comment on keeping the button white!

File: pegasus/sites.v3/advocacy.code.org/public/index.haml (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/advocacy.code.org/public/index.haml:74]: 😭

Comment (kelbyhawn) [reply pegasus/sites.v3/advocacy.code.org/public/index.haml:74]: Just checked Figma, and this mock is using an outdated button component that doesn't have a white option. This is too little too late, but I put an [updated button](https://www.figma.com/file/hNBxdgZ40KH4UZFG7ZEVo8/AI-Landing-Page-Updates---Spring-2023?type=design&node-id=3765%3A3010&mode=design&t=2GsNUuLjnVEeKH0r-1) component next to it using the white button.

Comment (hannahbergam) [reply pegasus/sites.v3/advocacy.code.org/public/index.haml:74]: I switched it back to the white button and updated the screenshot! Resolving:)

**Review by kelbyhawn — APPROVED** [review]

**Pull request merged**